src/AppBundle/Controller/SecurityController.php

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
Deciding only on the number of line of code is dangerous. As discussed in https://github.com/symfony/symfony/issues/21623, the bigger issue with `$container->get('foo')` is that you get no guarantee whatsoever about what you'll get back - no contrat. It's working "by luck" in terms of design.